README.md

```diff
@@ -12,6 +12,16 @@ For a list of examples please visit https://docs.edgexfoundry.org and navigate t
 
 Please see the [edgex-go README](https://github.com/edgexfoundry/edgex-go/blob/main/README.md).
 
+## Build Instructions
+
+#### Running the app service configurable service with docker
+
+app service configurable's docker images can be launched with docker-compose. Please refer to [EdgeX Docker Compose](https://github.com/edgexfoundry/edgex-compose/blob/main/README.md).
+
+#### Running the edgex app service configurable service as a snap
+
+For details on the EdgeX App Service Configurable Snap, including installation, configuration, please refer to [EdgeX App Service Configurable Snap](https://github.com/edgexfoundry/app-service-configurable/tree/main/snap).
+
```

Comment:
Let's put this on hold until we agree on a consistent way to document reference the packaging details: https://github.com/edgexfoundry/device-mqtt-go/pull/330

Reply:
Updated, thanks!